hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,50 +1,24 @@
 # USB mass storage boot
 
-**Available on Raspberry Pi 2B v1.2, 3A+, 3B, 3B+, 4B and 400 only.**
+**Available on Raspberry Pi 2B v1.2, 3A+, 3B, 3B+, and 4B only.**
```

Comment:
I suspect the 400 should be kept here, as the Pi 400 **does** support USB boot.

Reply:
Yes, 400 and CM4 support USB boot. I can pull in the corrections tomorrow when I get chance.

Reply:
done.

Reply:
> Yes, 400 and CM4 support USB boot. I can pull in the corrections tomorrow when I get chance.

CM3 Lite will do USB MSD boot I believe too - not sure about CM3 with onboard eMMC.

Reply:
I _think_ the CM3 might try USB MSD boot if there's no `bootcode.bin` on the boot partition of the eMMC?